txs/payload/bond_tx.go

```diff
@@ -5,12 +5,13 @@ import (
 
 	"github.com/hyperledger/burrow/acm/acmstate"
 	"github.com/hyperledger/burrow/crypto"
+	"github.com/hyperledger/burrow/genesis/spec"
 )
 
 func NewBondTx(pubkey crypto.PublicKey) (*BondTx, error) {
 	return &BondTx{
-		Inputs:   []*TxInput{},
-		UnbondTo: []*TxOutput{},
+		Input:     &TxInput{},
+		Validator: &spec.TemplateAccount{},
```

Comment:
This should be a `PublicKey`

Perhaps the field should be `ValidatorPublicKey` for clarity (in JS at least)